### ISSUE TITLE
Get the playground up-and-running on https://playground.vrl.dev

### ISSUE THREAD

**Issue by JeanMertz** [vectordotdev/vector#14711]
While we eventually want to allow people running the playground on their local machine (see stretch goals in #14652), we first intend to host our own playground at `https://playground.vrl.dev`.

We should consult the relevant parties to find out how/where we wish to host the playground, but _one_ potential solution is:

1. Create a new Netlify project for the playground.
2. Set up CI steps to compile/deploy the playground to Netlify.
3. Have `https://playground.vrl.dev` point to the new Netlify project.
4. Update main https://vrl.dev docs to reference and link to the playground.

**Comment by JeanMertz** [comment]
Reopening this, as it appears the deployment isn't working as expected yet:

https://github.com/vectordotdev/vector/pull/14905#issuecomment-1290210356

**Comment by JeanMertz** [comment]
Also just to check, did we update the docs on vrl.dev to point people to the new playground? (see point 4 above)

**Comment by jszwedko** [comment]
> Also, just to check, did we update the docs on vrl.dev to point people to the new playground? (see point 4 above)

I'd suggest we split that item off into a new issue. I don't really want to point users to it until it has a little more polish, but it is useful to have it deployed for the team to play with, give feedback, and share with individual users. I'm not saying the UI has to be finished first, just a little more polished.

**Comment by JeanMertz** [comment]
> I'm not saying the UI has to be finished first, just a little more polished.

Yeah, that's a fair point, we can track that separately.
